Localization/ko-KR/about.yaml

```diff
@@ -0,0 +1,7 @@
+--- 
```

Comment:
Even though this passes through a YAML parser, it doesn't quite follow the coding style we have for other YAML files. 

1. Could you stick to the style of the other YAML files, please? 
2. Could you keep the original comments in, please? They serve as metadata for non-Korean speakers to figure out what we are looking at. However feel free to add more comments.

Reply:
I see. I think it was my misunderstood. I thought I need to parse the file and commit it as parsed. I will make changes and request soon,

Reply:
Thanks, I will mark this as resolved. :)